package.json

```diff
@@ -125,6 +125,7 @@
     "nyc": "^13.0.0",
     "postcss": "^7.0.0",
     "prettier": "^1.8.2",
+    "prop-types-extra": "^1.1.0",
```

Comment:
This package isn't bundle size friendly, for instance:
https://github.com/mui-org/material-ui/blob/965776002fcf482b65d66040393cf75caa18d356/packages/material-ui/src/utils/exactProp.js#L9-L12. What do you think of inlining it as we did in v0.x?

Reply:
I didn't really look into but isn't the hole `propTypes` object removed in production i.e. it doesn't matter how many dependencies `propTypes` pulls in because they will be stripped?

Reply:
The imports aren't striped at least not in the wrap mode.